### PR TITLE
129 fix delete buttons on edit basket edit group edit items

### LINKED ISSUE #129: Fix Delete Buttons on edit basket, edit group, edit items
**User Story**
User story goes here

**Description**

**Acceptance Criteria**

Steps:

- step 1
- step 2

**Additional Information**

### PR DESCRIPTION
**Pull request by zmattes04**
## Developer: Zach Mattes
Closes #129 
Fixed delete buttons 
Everything works please lets merge it
Me and Jason worked on this
### Pull Request Summary

{Describe the purpose of your pull request}

### Modifications

{list out the files created/modified and a brief description of what was changed}

### Testing Considerations

{list out what you have tested and what the reviewer should verify}

### Pull Request Checklist

- [ ] Code is neat, readable, and works
- [ ] Comments are appropriate
- [ ] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [ ] The developer name is specified
- [ ] The summary is completed
- [ ] Assign reviewers

### Screenshots/Screencast

{put screenshots of your change, or even better a screencast displaying the functionality}

### CODE REVIEW

**Review by jhcsdev — COMMENTED** [review]
Looks good to me I only have one question which is wondering why we're using the ObjectId everywhere imported from mongoose instead of a string. Is there a benefit/something to be gained by using ObjectId? Im not against it at all I've just noticed it a lot and it seems more difficult to use imo but that could be just because I don't know what I'm looking at lol

File: frontend/src/pages/LoginPage.tsx
Comment (jhcsdev) [inline frontend/src/pages/LoginPage.tsx:1]: can we remove these console logs at some point also they're lowkey jankin up my console straight slonkin my chrome ngl

File: frontend/src/pages/MyGroupsPage.tsx
Comment (jhcsdev) [inline frontend/src/pages/MyGroupsPage.tsx:1]: see comment above we have so many of them

File: frontend/src/pages/LoginPage.tsx
Comment (SilveerDusk) [reply frontend/src/pages/LoginPage.tsx:1]: Facts, maybe instead of console logging on every keystroke we can only console log on submit?

**Comment by SilveerDusk** [comment]
> Looks good to me I only have one question which is wondering why we're using the ObjectId everywhere imported from mongoose instead of a string. Is there a benefit/something to be gained by using ObjectId? Im not against it at all I've just noticed it a lot and it seems more difficult to use imo but that could be just because I don't know what I'm looking at lol

The schema we have in the backend and what mongodb returns is a objectId, we have just been casting it to be a string, which is fine but I figured being true to its type would be better? Up for discussion tho, cause it can be a pain.

File: frontend/src/pages/LoginPage.tsx
Comment (jhcsdev) [reply frontend/src/pages/LoginPage.tsx:1]: Yeah i like that

**Pull request merged**